app/components/placement/summary_component.html.erb

```diff
@@ -9,9 +9,12 @@
     </h2>
   </div>
 
-  <div class="govuk-grid-column-one-quarter">
-    <%= render Placement::StatusTagComponent.new(placement:) %>
-  </div>
+  <dl class="app-result-detail">
+    <div class="app-result-detail__row">
+      <%= render Placement::StatusTagComponent.new(placement:) %>
+    </div>
+  </dl>
```

Comment:
From a HTML semantics point of view, I think this would be better as a `<div>`. We should only use a `<dl>` when we have `<dt>` and `<dd>` child elements inside.

```suggestion
  <div class="app-result-detail">
    <%= render Placement::StatusTagComponent.new(placement:) %>
  </div>
```

Reply:
Resolved in [3857392](https://github.com/DFE-Digital/itt-mentor-services/pull/689/commits/3857392603bb84909be6d41dccc73a3fcfe0c35a)